src/Test/Utilities/Portable/Diagnostics/DiagnosticDescription.cs

```diff
@@ -212,12 +217,13 @@ public DiagnosticDescription WithLocation(int line, int column)
         /// <param name="syntaxPredicate">The argument to syntaxPredicate will be the nearest SyntaxNode whose Span contains first squiggled character.</param>
         public DiagnosticDescription WhereSyntax(Func<SyntaxNode, bool> syntaxPredicate)
         {
-            return new DiagnosticDescription(_code, _isWarningAsError, _squiggledText, _arguments, _startPosition, syntaxPredicate, _argumentOrderDoesNotMatter, _errorCodeType, _defaultSeverityOpt, _effectiveSeverityOpt);
+            return new DiagnosticDescription(_code, _isWarningAsError, _squiggledText, _arguments, _startPosition, syntaxPredicate, _argumentOrderDoesNotMatter, _errorCodeType, _defaultSeverityOpt, _effectiveSeverityOpt, _isSuppressed);
         }
 
         public object Code => _code;
         public bool HasLocation => _startPosition != null;
         public bool IsWarningAsError => _isWarningAsError;
+        public bool IsSuppressed => _isSuppressed;
```

Comment:
`IsSuppressed` should be taken into account in the below `Equals` method.

Reply:
It should also be taken into account in `GetHashCode` and `ToString` methods

Reply:
Fixed.